command/snapshot/save/snapshot_save.go

```diff
@@ -23,17 +26,26 @@ func New(ui cli.Ui) *cmd {
 }
 
 type cmd struct {
-	UI    cli.Ui
-	flags *flag.FlagSet
-	http  *flags.HTTPFlags
-	help  string
+	UI                 cli.Ui
+	flags              *flag.FlagSet
+	http               *flags.HTTPFlags
+	help               string
+	appendFileNameFlag flags.StringValue
+}
+
+func (c *cmd) getAppendFileNameFlag() *flag.FlagSet {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	fs.Var(&c.appendFileNameFlag, "append-filename", "Append filename flag takes two possible values. "+
```

Comment:
This description is a bit confusing to me. Can we rephrase this to something like?
```
Append filename flag supports the following comma-separated arguments. 1. version, 2. dc. 3. node 4. status. It appends these values to the filename provided in the command
```